src/ext/resources/legacygraphqlapi/schema.graphqls

```diff
@@ -1236,28 +1236,63 @@ input InputUnpreferred {
 }
 
 enum RoutingErrorCode {
-    """No transit connection was found between the origin and destination withing the operating day or the next day"""
+    """
+    No transit connection was found between the origin and destination within the operating day or
+    the next day, not even sub-optimal ones.
+    """
     NO_TRANSIT_CONNECTION
 
-    """Transit connection was found, but it was outside the search window, see metadata for the next search window"""
+    """
+    A transit connection was found, but it was outside the search window. See the metadata for a token
+    for retrieving the result outside the search window.
+    """
     NO_TRANSIT_CONNECTION_IN_SEARCH_WINDOW
 
-    """The date specified is outside the range of data currently loaded into the system"""
+    """
+    The date specified is outside the range of data currently loaded into the system as it is too
+    far into the future or the past.
+
+    The specific date range of your system is configurable but also depends on the input data
+    provided.
+    """
     OUTSIDE_SERVICE_PERIOD
 
-    """The coordinates are outside the bounds of the data currently loaded into the system"""
+    """
+    The coordinates are outside the geographic bounds of the transit and street data currently loaded
+    into the system and therefore cannot return any results.
+    """
     OUTSIDE_BOUNDS
 
-    """The specified location is not close to any streets or transit stops"""
+    """
+    The specified location is not close to any streets or transit stops currently loaded into the
+    system, even though it is generally within its bounds.
+
+    This can happen when there is only transit but no street data coverage at the location in
+    question.
+    """
     LOCATION_NOT_FOUND
 
-    """No stops are reachable from the location specified. You can try searching using a different access or egress mode"""
+    """
+    No stops are reachable from the start or end locations specified.
+
+    You can try searching using a different access or egress mode, for example cycling instead of walking,
+    increase the walking/cycling/driving speed or change the system's configuration so that stops
```

Comment:
Again, only a small subset of API users can change the system's configuration.

Reply:
See above.